### DISCUSSION TITLE
Why network module in example/. define reset_parameters manually?

### DISCUSSION THREAD

**Discussion by PolarisRisingWar** [snap-stanford/ogb#227] [General]
I find in [ogb/gnn.py at master · snap-stanford/ogb](https://github.com/snap-stanford/ogb/blob/master/examples/nodeproppred/arxiv/gnn.py) that in the code `reset_parameters` function is defined manully. But I find that it just use default reset_parameters function of conv and bn again.  
I searched on Google and they said in Modules, parameters will automatically defined by `reset_parameters` function. So why should we reuse it again?

**Comment by rusty1s** [comment]
Inside the `model.reset_parameters()` function we reset the parameters of each sub-module since we want to train our model for a number of runs with different initializations.

**Reply by PolarisRisingWar** [reply]
Oh, I got it! Thank you very much!
